client.lua

```diff
@@ -191,13 +216,14 @@ local function startTestDriveTimer(testDriveTime, shop)
                         type = 'success'
                     })
                 end
-                drawTxt(Lang:t('general.testdrive_timer') .. math.ceil(testDriveTime - secondsLeft / 1000), 4, 0.5, 0.93, 0.50, 255, 255, 255, 180)
+                drawTxt(Lang:t('general.testdrive_timer') .. math.ceil(time - secondsLeft / 1000), 4, 0.5, 0.93, 0.50, 255, 255, 255, 180)
             end
             Wait(0)
         end
     end)
 end
 
+--- Zoning function. Happens upon entering any of the sell zone.
```

Comment:
Similar to an earlier comment, better to summarize the contents of the function rather than restate the name of the function.